assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -53,25 +52,28 @@ export default function WidgetAreaRenderer( { slug, totalAreas } ) {
 	// A combined CTA will span the combined width of all widgets that it was
 	// combined from.
 	const {
-		gridClassNames,
+		gridColumnWidths,
 		overrideComponents,
 	} = combineWidgets( widgets, widgetStates, {
-		classNames,
 		columnWidths,
 		rowIndexes,
 	} );
 
 	// Render all widgets.
 	const widgetsOutput = widgets.map( ( widget, i ) => (
-		<WidgetRenderer
-			gridClassName={ classnames( gridClassNames[ i ] ) }
-			OverrideComponent={ overrideComponents[ i ] ? () => {
-				const { Component, metadata } = overrideComponents[ i ];
-				return <Component { ...metadata } />;
-			} : undefined }
-			key={ widget.slug }
-			slug={ widget.slug }
-		/>
+		<WidgetWrapper
+			key={ `${ widget.slug }-wrapper` }
+			gridColumnWidth={ gridColumnWidths[ i ] }
+		>
+			<WidgetRenderer
+				OverrideComponent={ overrideComponents[ i ] ? () => {
+					const { Component, metadata } = overrideComponents[ i ];
+					return <Component { ...metadata } />;
+				} : undefined }
+				key={ widget.slug }
```

Comment:
This prop shouldn't be necessary anymore now that it's not the top component in the loop.